### PR TITLE
10.0 mig website sale suggest create account

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: website_sale_suggest_create_account/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* Log out.
+* Buy something from your website.
+* Go to your cart.
+
+There you will see the new options.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
```
Comment (yajo) [inline website_sale_suggest_create_account/README.rst:41]: 10.0

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
It seems to me that blue buttons should have a bit of margin by default:

captura de pantalla de 2016-11-02 14-47-20

**Comment by yajo** [comment]
Please fix the 404 from https://github.com/OCA/e-commerce/pull/138#discussion_r86314022

**Comment by pedrobaeza** [comment]
Superseeded by #172

**Pull request closed without merging**